doc/usermanual/overview/overview.xml

```diff
@@ -85,10 +85,9 @@
     </para></listitem>
 
     <listitem><para>
-      The powerful export system supports Google+ webalbum, flickr upload, disk storage, 1:1
-      copy, email attachments and can generate a simple html-based web gallery. darktable allows
-      you to export to low dynamic range (JPEG, JPEG2000, PNG, TIFF, PDF), 16-bit (PPM, TIFF),
-      or linear high dynamic range (PFM, EXR) images.
+      darktable can export developed images as low or high dynamic range files (JPEG, PNG, TIFF, 
+      PDF, PFM, EXR) to local disk storage, some web albums, LaTeX book template,
```

Comment:
"some web albums" sounds unspecific and/or dismissive to me. I'd go for "various third-party web albums", if you want to convey "yeah, we do stuff, but it may change depending on what said third-party do" :)

Reply:
to say "various", we need at least two :) as we seem to have only one working (piwigo), then I prefer to say " to... web albums"